app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -106,4 +106,8 @@ def textual_endpoints
      {:label => _('Hawkular API Port'),
       :value => @ems.connection_configurations.hawkular.endpoint.port}]
   end
+
+  def textual_meta_data
+    textual_key_value_group(@record.custom_attributes.select(:name, :value).to_a)
```

Comment:
We show custom attributes on other screens, so I would expect that we would use the same methods from those other screens for consistency.